src/utils.js

```diff
@@ -230,9 +232,8 @@ var Utils = exports.Utils = {
       return;
     }
     var disabledSites = JSON.parse(localStorage.disabledSites);
-    var idx = disabledSites.indexOf(origin);
-    if(idx >= 0){
-      Utils.removeElementFromArray(disabledSites, idx);
+    if(disabledSites[origin]){
+      delete disabledSites[origin];
```

Comment:
Efficiency is less of a concern here since this function would only be called infrequently. Still we should probably use a global object here instead of serializing from localStorage each time.